docs/examples/tips_and_tricks.rst

```diff
@@ -10,14 +10,20 @@ Tips & Tricks
 Including sunk costs for previous investments into specific assets
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-It is possible to include the previous investment costs into now pre-existing asset capacities in the economic evaluation of a scenario. For this, following trick can be applied to the input data of the asset in question, by setting:
+Usually, the investments into existing capacities are neglected and assumed to be sunk costs of the system. The existing capacity :code:`installedCap` as well as the age of the installed asset :code:`age_installed` are only used to calculate when necessary re-investments take place, and how high the :ref:`replacements costs <Cost_calculations>` are.
+
+When optimizing a system with pre-existing capacities of certain assets, it can be usefull for the user to implement the installation costs of these assets in the economic evaluation.
```

Comment:
it can be useful for what usecase? The formulation is rather theoretical, an example would be welcome to get an idea